sklego/pandas_utils.py

```diff
@@ -86,8 +89,10 @@ def wrapper(*args, **kwargs):
         func_args_str = "".join(
             ", {} = {!r}".format(*item) for item in list(func_args.items())[1:]
         )
-        print_fn(f"[{func.__name__}(df{func_args_str})] " + combined,)
-
+        if print_args:
+            print_fn(f"[{func.__name__}(df{func_args_str})] " + combined,)
+        else:
+            print_fn(f"[{func.__name__}()")
```

Comment:
The formatting here is a bit off. 
```suggestion
            print_fn(f"[{func.__name__}]" + combined,)
```